.github/workflows/blade-pr-title-check.yml

```diff
@@ -0,0 +1,40 @@
+name: Blade PR Title Check
+
+on: 
+  pull_request:
+     types: [open, edited, synchronize]
+
+concurrency: # cancel previously running workflows when a new workflow is re-run
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-title-check:
+    name: Check
+    runs-on: ubuntu-latest # nosemgrep: non-self-hosted-runner
```

Comment:
We're not running this on self hosted?

Reply:
No, I saw rest of our workflows aren't either. I can't recall why we made this decision though. @kamleshchandnani do you recall?

Reply:
nupe because this is public repo. I think we can't run self-hosted on public repo or they need to be configured. regardless i don't see advantages of why shall we run this on self-hosted